test-cases/tests/authorization/b04b-verify-dedicated-admin-user-permissions-are-correct.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Should [this test case](https://github.com/integr8ly/integreatly-operator/blob/5ccc7d3fa5954d2f673cb8f0f90ff0f9d3f206bf/test-cases/tests/authorization/b01b-verify-that-the-users-can-login-in-all-products.md) be updated instead?

Reply:
Good catch, completely updated the wrong test case :sweat_smile: